### ISSUE TITLE
Unable to read detailed information in the "processes" tab of Windows 10 Task manager

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#7148]
Tested with Windows 10 Insider build 16188
NVDA version next-14027,c80e529f

Starting with insider builds after the main RS2 release, NVDA is unable to read detailed information about processes in the task manager. This works with Narrator review commands.

* Expected behavior: NVDA should be able to read columns such as processor utilization, memory, network, etc.
* Actual behavior: NVDA only reads  process name with any navigation.
* Ideal behavior: NVDA should automatically read columns when arrowing down. Object navigation should not be necessary.

See log for details. Possibly related to or duplicate of #2752.

STR:
1. Go to Windows Task Manager by pressing control+shift+escape.
2. Focus on the processes tab.
3. Tab to the details button and activate it if necessary.
4. In the processes tab, go to any process and attempt to obtain details about the process  using any review commands including object nav.

Log:
Speaking [LangChangeCommand ('en'), u'Background processes  grouping']
IO - speech.speak (01:47:51):
Speaking [LangChangeCommand ('en'), u'Process: CrashPlan backup engine which backs up your files!  expanded  row 15  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:47:56):
Input: kb(laptop):shift+NVDA+downArrow
IO - speech.speak (01:47:56):
Speaking [LangChangeCommand ('en'), u'Service: CrashPlan Backup Service  not selected  row 1  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:47:57):
Input: kb(laptop):shift+NVDA+rightArrow
IO - speech.speak (01:47:57):
Speaking [LangChangeCommand ('en'), u'No next']
IO - inputCore.InputManager.executeGesture (01:48:40):
Input: kb(laptop):leftArrow
IO - speech.speak (01:48:40):
Speaking [LangChangeCommand ('en'), u'collapsed']
IO - inputCore.InputManager.executeGesture (01:48:40):
Input: kb(laptop):shift+NVDA+downArrow
IO - speech.speak (01:48:40):
Speaking [LangChangeCommand ('en'), u'No objects inside']
IO - inputCore.InputManager.executeGesture (01:48:42):
Input: kb(laptop):rightArrow
IO - speech.speak (01:48:42):
Speaking [LangChangeCommand ('en'), u'expanded']
IO - inputCore.InputManager.executeGesture (01:48:42):
Input: kb(laptop):shift+NVDA+downArrow
IO - speech.speak (01:48:42):
Speaking [LangChangeCommand ('en'), u'No objects inside']
IO - inputCore.InputManager.executeGesture (01:48:43):
Input: kb(laptop):leftArrow
IO - speech.speak (01:48:43):
Speaking [LangChangeCommand ('en'), u'collapsed']
IO - inputCore.InputManager.executeGesture (01:48:44):
Input: kb(laptop):upArrow
IO - speech.speak (01:48:44):
Speaking [LangChangeCommand ('en'), u'Process: Cortana  row 14  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:48:45):
Input: kb(laptop):downArrow
IO - speech.speak (01:48:45):
Speaking [LangChangeCommand ('en'), u'Process: CrashPlan backup engine which backs up your files!  collapsed  row 15  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:48:45):
Input: kb(laptop):shift+NVDA+downArrow
IO - speech.speak (01:48:46):
Speaking [LangChangeCommand ('en'), u'Service: CrashPlan Backup Service  not selected  row 1  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:48:47):
Input: kb(laptop):shift+NVDA+rightArrow
IO - speech.speak (01:48:47):
Speaking [LangChangeCommand ('en'), u'No next']
IO - inputCore.InputManager.executeGesture (01:48:50):
Input: kb(laptop):shift+NVDA+rightArrow
IO - speech.speak (01:48:50):
Speaking [LangChangeCommand ('en'), u'No next']
IO - inputCore.InputManager.executeGesture (01:48:51):
Input: kb(laptop):upArrow
IO - speech.speak (01:48:51):
Speaking [LangChangeCommand ('en'), u'Process: Cortana  row 14  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:48:52):
Input: kb(laptop):downArrow
IO - speech.speak (01:48:52):
Speaking [LangChangeCommand ('en'), u'Process: CrashPlan backup engine which backs up your files!  collapsed  row 15  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:48:52):
Input: kb(laptop):shift+NVDA+rightArrow
IO - speech.speak (01:48:52):
Speaking [LangChangeCommand ('en'), u'Process: dirmngr.exe (32 bit)  collapsed  not selected  row 16  column 1  level 1']
IO - inputCore.InputManager.executeGesture (01:48:54):
Input: kb(laptop):shift+NVDA+leftArrow
IO - speech.speak (01:48:54):
Speaking [LangChangeCommand ('en'), u'Process: CrashPlan backup engine which backs up your files!  collapsed  row 15  column 1  level 1']

**Comment by josephsl** [comment]
Hi, confirmed in latest fast ring and build 15063. The recommended workaround for now is to temporarily turn off simple review mode. Thanks.

**Comment by jcsteh** [comment]
This is a regression caused by #7019, which was needed to fix #6948 and #6950 (extraneous objects in review in Settings, Edge, etc.). We now only consider UIA objects which have been explicitly marked as both control and content. Microsoft has marked these objects as being neither. While this is arguably a "regression" in NVDA, the new behaviour is correct according to specifications and allows authors to control what is relevant. I think this is something that needs to be fixed by Microsoft.

**Comment by PratikP1** [comment]
Thank you, @josephsl, for the temporary work around suggestion. I’ll report to Microsoft through my channel.

**Comment by PratikP1** [comment]
Based on some discussion, I've added two feedback items for Windows 10 that will help mitigate this issue + other keyboard navigation issues. Here are the links for the two items.

https://aka.ms/mz8q1v

https://aka.ms/u7uj60

**Comment by Mohamed00** [comment]
Shouldn't this have the NeedsExternalFix label?

**Comment by ehollig** [comment]
Label added. @PratikP1, any updates on this issue?

**Comment by Mohamed00** [comment]
While I'm not @PratikP1, as of Windows 10 Insider build 16281.1000, the issue reported in this ticket is still occurring.

**Comment by PratikP1** [comment]
The problems reported in this ticket still exist as of Win 10 build 16288 with the latest NVDA next. Microsoft is still examining the issue from their end as well.

**Comment by Neurrone** [comment]
Still happening on 1709 RTM.

**Comment by Neurrone** [comment]
@PratikP1 any news from Microsoft?

**Comment by PratikP1** [comment]
I’ve asked for an update. I will provide details as soon as I hear back.

**Comment by Neurrone** [comment]
Although this is technically not a solution, [installing the windows 7 task manager files](https://winaero.com/blog/get-classic-old-task-manager-in-windows-10/) works very well - no lag, and perfectly accessible.

**Comment by Mohamed00** [comment]
Since a workaround for this issue is in the latest version of NVDA, this issue should probably be closed.

**Comment by derekriemer** [comment]
Fixed in #8147 although microsoft should really still fix themselves.
